internal/pkg/plugin/jenkins/precreate.go

```diff
@@ -16,29 +14,28 @@ import (
 )
 
 const (
-	JenkinsName          = "jenkins"
-	JenkinsNamespace     = "jenkins"
-	JenkinsDataDirectory = "/data/jenkins-volume/"
-	JenkinsUid           = 1000
-	JenkinsGid           = 1000
+	JenkinsName                      = "jenkins"
+	JenkinsNamespace                 = "jenkins"
+	JenkinsDataDirectory             = "/data/jenkins-volumes/"
```

Comment:
`/data/jenkins-volume` is better. It's a single volume.

Reply:
This is more precise, I agree with you.